autonomy/cli/develop.py

```diff
@@ -64,3 +91,31 @@ def run_service_locally(image: str) -> None:
 
     click.echo("Stopping container.")
     container.stop()
+
+
+@develop_group.group("publish")
```

Comment:
“autonomy develop publish” is a very suboptimal naming choice imo. The command group should be “register” or “mint”

Reply:
I agree, I wanted to ask you for this once this PR was ready for review